index.js

```diff
@@ -0,0 +1,41 @@
+import { DynamoDB } from "@aws-sdk/client-dynamodb"
+
+const node_env = process.env.NODE_ENV;
+const setup_db = process.env.SETUP_DB;
+
+if ((node_env !== 'production') && (setup_db==='true')) {
+    const client = new DynamoDB({endpoint: "http://localhost:8000"});
+
+//Create a table
```

Comment:
Leaving comments in code is useful only if you think what you are doing is difficult to udnerstand. In this case we might not need it.